SwiftUI_DZ/SwiftUI_DZ/ContentView.swift

```diff
@@ -8,14 +8,91 @@
 import SwiftUI
 
 struct ContentView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let showBalanceString = "Показать баланс"
+        static let cashString = "На счету -- "
+        static let cashInt = 300
+        static let replenishString = "Пополнить счет"
+        static let bankScoreString = "Банковский счет"
+        static let addCashString = "Пополнено успешно!"
+        static let okString = "OK"
+        static let walletString = "Кошелек"
+        static let openScoreString = "Открыть счет"
+        static let scoreOpenedString = "Счет открыт"
+        static let transferString = "Перевести средства"
+        static let sureString = "Вы уверенны?"
+        static let cardString = "creditcard"
+        static let scoreInt = 1000
+        static let offsetXCGDouble: Double = 400
+        static let fontSize = 30.0
+        static let zeroDouble = 0.0
+        static let cornerRadiusDouble = 10.0
+    }
+    
+    // MARK: - Private Properties
+    
+    @State private var isShowBalance = false
+    @State private var isReplenish = false
```

Comment:
```suggestion
    @State private var isReplenish = false
```
Все приватки ниже var body